From f0de38394feba36de5448d11fe57b674dbf1f946 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <uwu@icenowy.me>
Date: Tue, 27 Feb 2024 20:38:22 +0800
Subject: [PATCH 1/2] arm64: dts: qcom: sdm845-xiaomi-polaris: Enable ath10k
 host-cap skip quirk

The WiFi firmware used on Polaris (Xiaomi Mi Mix2s) doesn't
support the host-capability QMI request, hence enable the skip
quirk for this device.

Signed-off-by: Icenowy Zheng <uwu@icenowy.me>
---
 arch/arm64/boot/dts/qcom/sdm845-xiaomi-polaris.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-xiaomi-polaris.dts b/arch/arm64/boot/dts/qcom/sdm845-xiaomi-polaris.dts
index e386b504e978b..8df61887e3dc4 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-xiaomi-polaris.dts
+++ b/arch/arm64/boot/dts/qcom/sdm845-xiaomi-polaris.dts
@@ -711,5 +711,6 @@ &wifi {
 	vdd-1.3-rfa-supply = <&vreg_l17a_1p3>;
 	vdd-3.3-ch0-supply = <&vreg_l25a_3p3>;
 	vdd-3.3-ch1-supply = <&vreg_l23a_3p3>;
+	qcom,snoc-host-cap-skip-quirk;
 	status = "okay";
 };
-- 
2.43.0

